### PR TITLE
feat: remove banner and update homepage

### LINKED ISSUE #1594: Enhancement[Homepage]: Design update including Banner feature tile
## Enhancement[Homepage]: Design update including Banner feature tile

See design specs:
https://github.com/carbon-design-system/carbon-website/issues/1424

- [x] Add a feature tile on the homepage banner to accommodate "Start migrating" content; card similar to "Start Designing" and "Start Developing" 

- [x] Card replaces redundant "Carbon Design System" text with a callout to migration; in turn, change "Inclusive, flexible and consistent" to "Carbon Design System"

- [x] Move pause button to accommodate new banner tile and to align with other sites' pause location

- [x] Adjust aspect ratio of all tiles (every tile and featured article image should have an aspect ratio of 2:1 -- no 16:9s (see geometry specs at all breakpoints)

- [x] Adjust vertical height of Start Designing and Start Developing images specific to breakpoints so they don't look wonky while scaling responsively (see geometry specs at all breakpoints)

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #1594
closes https://github.com/carbon-design-system/carbon-website/issues/1735
Ref https://github.com/carbon-design-system/carbon-website/issues/1424 (design spec)

Homepage design enhancements, including removing blue alert banner and adding a new tile.

#### Changelog

**New**

- homepage tile

**Changed**

- Updated homepage text.
- moved pause video button
- mobile/tablet styles for feature tile
- fix for height of side nav
- fix for anchor links

**Removed**

- Blue alert banner at top

## ToDo

- [x] Update feature tile images at different breakpoints

### CODE REVIEW

File: src/components/Homepage/Homepage.js [hunk 1]
```diff
@@ -2,39 +2,30 @@ import React from 'react';
...
       <div>
         <span className="homepage--dots" />
         <section className="homepage--header">
           <div className="bx--grid">
             <div className="bx--row">
-              <div className="bx--col-lg-4 bx--offset-lg-4">{homeTitle}</div>
+              <div className="bx--col-lg-4 bx--col-md-4 bx--col-sm-2 bx--offset-lg-12 bx--offset-md-4 bx--offset-sm-2 homepage--tile-header">
+                <ClickableTile
+                  title="Read"
+                  titleLarge="Migrate from v9 to v10"
```
Comment (connor-leech) [inline src/components/Homepage/Homepage.js]:
```suggestion
                  titleLarge="Migrate to v10"
```

**Comment by jeanservaas** [comment]
Hey @alisonjoseph thanks for picking this up!!

- At Xlg breakpoint, looks like the pause button is too far off the left menu right now, it should be 16px off the side nav and 16px up from the carbon design system band. basically it should just totally mimic the pause button positioning on the IDL site.

image

image


- Also, in mobile, there shouldn't be a margin on the banner feature square

<br>

preview link

image


<br>

spec
image

<br>

- Feature tile should be the same size and aspect ratio as the start designing and start developing tiles -- right now feature tile looks smaller

<br>

preview link

image

<br>

spec

image


<br>

- At tablet breakpoint the start designing and start developing tiles should only be 320px tall (i.e. have a 2:1 aspect ratio) -- they look much taller in the preview link for some reason and the >_ is getting really small and getting pushed off of vertical center. Also there should be 48px between the bottom of the Carbon Design System banner and the top of the Getting Started Design tile and 16px gutters (not 24px) between all images on the homepage


image


- @shixiedesign brought this up on interior pages — need to adjust anchor links to account for missing blue bar (i.e. jog up position of anchor links 

image

**Comment by alisonjoseph** [comment]
Quick question about this comment

> At tablet breakpoint the start designing and start developing tiles should only be 320px tall (i.e. have a 2:1 aspect ratio) 

If they are a set height, then they won't be a set aspect ratio because the width varies. They are a set height right now, but I think it is taller than 320px. I can adjust to just be 2:1 aspect ratio, I think thats what the spec is trying to say

**Comment by alisonjoseph** [comment]
@jeanservaas I've made some updates.

- fixed video height at mobile breakpoint so it is 2x the square tile. 
- removed 1rem padding on left/right of square tile at mobile breakpoint
- updated spacing above/below the "getting started" tiles to be 48px/80px
- updated tablet sizing for "getting started" tiles to be set at 320px tall

File: src/styles/_sticky.scss [hunk 2]
```diff
@@ -89,6 +89,7 @@ website global header hidden
   // side nav visible, website alert not present
   .#{$prefix}--header ~ .#{$prefix}--side-nav {
     transform: translate(0, -3rem);
+    height: 100%;
```
Comment (tw15egan) [inline src/styles/_sticky.scss]: Thank you for fixing this 🙏

**Comment by alisonjoseph** [comment]
Don't merge until @jeanservaas approves pls 😄  not 100% sure if we have the tablet "getting started blocks" correct yet

File: src/styles/_sticky.scss (hunk 2 above)
Comment (alisonjoseph) [reply src/styles/_sticky.scss]: Yesss its been driving me crazy! 👍

**Comment by jeanservaas** [comment]
Looking good, just a few more things!

- Just need to get the feature card and the start designing and start developing card the same size and dimensions at every breakpoint but mobile and also the same type tokens should be used (Migrate to v10 looks like a smaller type size at the Xlg breakpoint)

image

- Also, can you change this margin (between start designing / start developing) to 16px from 24px?

image


- Just noticed this but arrow colors and spacing also look different on feature card... banner feature card arrow looks too light (maybe it's using white instead of f3f3f3

image

- Anchor link distance still seems to be happening (i.e. assuming the 40px blue banner is still there when you click to jump down the page. Again, this was a Shixie catch, so lmk if what I'm saying doesn't make sense here.

image


Thanks girl!!!

**Comment by alisonjoseph** [comment]
- icons are all gray 10 
- card sizes should match now 
- larger title type is using the same tokens (productive heading 03)
- spacing between tiles is 16px/1rem
- updated anchor link offset from top (this one is a little tricky because some pages have tabs and others don't so I picked a value that works for both, we can look at updating based on whether a page has tabs or not in the theme or a future issue/PR)

Hopefully, this is it @jeanservaas 🙏

**Pull request merged**